app/src/common/shared/org/mozilla/vrbrowser/browser/SessionStore.java

```diff
@@ -991,6 +991,16 @@ private String checkYoutubeOverride(String aUri) {
             String result = aUri;
             result += aUri.contains("?") ? "&" : "?";
             result += "disable_polymer=1";
+            if (uri.getQueryParameter("app") == null) {
+                result += "&app=desktop";
+            }
+            if (uri.getQueryParameter("persist_app") == null) {
+                result += "&persist_app=1";
+            }
+            if (uri.getQueryParameter("no_app") == null) {
```

Comment:
typo: there should not be an underscore in this one: `no_app` -> `noapp`

<hr>

https://www.youtube.com/watch?v=sPyAQQklc1s
becomes →
https://www.youtube.com/watch?v=sPyAQQklc1s&app=desktop&persist_app=1&noapp=1
becomes →
https://www.youtube.com/watch?v=sPyAQQklc1s

Reply:
Done